Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml

```diff
@@ -13,14 +14,90 @@
         <Grid.Resources>
             <DataTemplate x:Key="IsSelectedTemplate">
                 <CheckBox x:Name="IsSelected_FieldSelection" Click="IsSelected_FieldSelection_Click" Style="{StaticResource @GridCellCheckBoxStyle}" IsEnabled="{Binding IsNotMandatory, Mode=TwoWay,UpdateSourceTrigger=PropertyChanged}" IsChecked="{Binding IsSelected,Mode=TwoWay,UpdateSourceTrigger=PropertyChanged}" VerticalAlignment="Top" HorizontalAlignment="Center"/>
+                <DataTemplate.Triggers>
+                    <DataTrigger Binding="{Binding defaultOperation}" Value="Do Not Add New">
+                        <Setter Property="IsEnabled" TargetName="IsSelected_FieldSelection" Value="False" />
+                        <Setter Property="IsChecked" TargetName="IsSelected_FieldSelection" Value="False" />
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding defaultOperation}" Value="Add New">
+                        <Setter Property="IsChecked" TargetName="IsSelected_FieldSelection" Value="True" />
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding defaultOperation}" Value="Replace Existing with New">
+                        <Setter Property="IsChecked" TargetName="IsSelected_FieldSelection" Value="True" />
+                    </DataTrigger>
+                </DataTemplate.Triggers>
             </DataTemplate>
+
+            <DataTemplate x:Key="xMatchingModelTemplate">
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="120*" />
+                        <ColumnDefinition Width="50" />
+                    </Grid.ColumnDefinitions>
+
+                    <TextBlock x:Name="xMatchingAPIName" ToolTip="{Binding MatchingAPIName}" Text="{Binding MatchingAPIName, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" Grid.Column="0" />
+                    <UserControls:ucButton x:Name="xManualMatchBtn" VerticalAlignment="Center" IsEnabled="True" Grid.Column="1" Margin="0" Padding="0" Height="23"
+                      ButtonType="ImageButton" Click="XManualMatchBtn_Click" ButtonText="" ToolTip="Manually Match an Existing API Model" ButtonFontImageSize="15" ButtonImageHeight="15"
+                            ButtonImageWidth="15" ButtonImageType="Edit" />
+                </Grid>
+            </DataTemplate>
+
+            <DataTemplate x:Key="CompareAndMergeTemplate">
```

Comment:
missing staring 'x' here and other element in this page xaml

Reply:
Done !